src/HyperFormula.ts

```diff
@@ -1423,8 +1463,9 @@ export class HyperFormula implements TypedEmitter {
    * Checks if it possible to reorder columns of a sheet according to a permutation.
    *
    * @param {number} sheetId - ID of a sheet to operate on
-   * @param {number[]} newColumnOrder - permutation of columns
+   * @param {number[]} newRowOrder - permutation of columns
```

Comment:
Good catch but method is called `isItPossibleToSetColumnOrder`, TSDoc is correct, it's the argument name that is wrong